src/components/DiscordVerification.tsx

```diff
@@ -322,8 +316,15 @@ const DiscordVerification: React.FC<DiscordVerificationProps> = ({
             </Text>
             <Text fontSize="sm" color="gray.600">
               Paste and send the command in any channel, and then enter your
-              username
+              username in the popup
```

Comment:
Same here. Maybe use something along the lines of "discord username" and "swecc username" to differentiate between the two.